var/spack/repos/builtin/packages/rose/add_spack_compiler_recognition.patch

```diff
@@ -1,13 +0,0 @@
-diff --git a/config/compiler-defs.m4 b/config/compiler-defs.m4
```

Comment:
Is there a reason you are modifying patches in other packages? Could you update the PR description to explain why?